pkg/ibm/vpc.go

```diff
@@ -111,6 +112,15 @@ func (c *CloudClient) TerminateVPC(vpcID string) error {
 	if err != nil {
 		return err
 	}
+	// wait until vpc is deleted
+	for {
+		_, err = c.GetVPCByID(vpcID)
+		if err != nil {
+			break
+		}
+		time.Sleep(1 * time.Second)
+		fmt.Printf("Waiting for VPC to be deleted..\n")
+	}
```

Comment:
@praveingk can you please add a `maxAttempts`/`timeLimit` parameter to avoid an endless loop (resources deletion processes could hang for a long time on IBM cloud). Upon reaching the threshold value the method should return an error.
For example see `attachTag()` in `tagging.go`.